src/components/molecules/Settings/Project/Plugin/PluginSection/index.tsx

```diff
@@ -21,25 +31,71 @@ export type PluginPageMode = "list" | "install-way" | PluginInstallWay;
 
 export type PluginInstallWay = "install-zip" | "install-public-repo" | "install-private-repo";
 
+export type PluginTabs = "Library" | "Installed" | "Uploaded";
+
 const PluginSection: React.FC<Props> = ({
   loading,
   installedPlugins,
+  extensions,
+  accessToken,
   installByUploadingZipFile,
   installFromPublicRepo,
   uninstallPlugin,
 }) => {
+  const t = useT();
+  const { search } = useLocation();
+  const queriedPluginId = useMemo(
+    () => new URLSearchParams(search).get("pluginId") ?? undefined,
+    [search],
+  );
+
+  console.log(queriedPluginId, "aldsfkjalskdfljksdf");
+
+  const tabHeaders = {
+    Library: t("Plugin Library"),
+    Installed: t("Installed"),
+    Uploaded: t("Uploaded Plugins"),
+  };
   return (
     <>
-      {loading ? (
-        <Loading />
-      ) : (
-        <PluginInstall
-          installedPlugins={installedPlugins}
-          installFromPublicRepo={installFromPublicRepo}
-          installByUploadingZipFile={installByUploadingZipFile}
-          uninstallPlugin={uninstallPlugin}
-        />
-      )}
+      <TabSection<PluginTabs> selected="Library" menuAlignment="top" headers={tabHeaders}>
+        {{
+          Library: (
+            <Box p="2xl">
+              {extensions?.library?.map(ext => (
+                <ext.component
+                  key={ext.id}
+                  pluginId={queriedPluginId}
+                  accessToken={accessToken}
+                  onUninstall={uninstallPlugin}
```

Comment:
onInstall should be set also

Reply:
As I said on Slack, I was waiting on your implementation for Marketplace installation since I didn't know if we would use the same mutation as installing from github, or from local Zip, or a new mutation. 
If that implementation is complete, please let me know which install mutation to use and I'll add it

Reply:
Already done. Use `installPlugin` mutation. pluginId should be `xxxxx@1.0.0` (`name@version`), for example. Automatically Re:Earth finds plugins from martketplace